test/bpf_ut/bpftest/workload_test.go

```diff
@@ -0,0 +1,281 @@
+//go:build linux && (amd64 || arm64) && !aix && !ppc64
+
+/*
+ * Copyright The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bpftests
+
+import (
+	"testing"
+
+	"github.com/cilium/ebpf"
+
+	"kmesh.net/kmesh/api/v2/workloadapi/security"
+	bpf2go "kmesh.net/kmesh/bpf/kmesh/bpf2go/dualengine"
+	"kmesh.net/kmesh/daemon/options"
+	"kmesh.net/kmesh/pkg/bpf/factory"
+	bpfUtils "kmesh.net/kmesh/pkg/bpf/utils"
+	bpfWorkload "kmesh.net/kmesh/pkg/bpf/workload"
+	"kmesh.net/kmesh/pkg/cache/v2/maps"
+	"kmesh.net/kmesh/pkg/constants"
+	controllerWorkload "kmesh.net/kmesh/pkg/controller/workload"
+	"kmesh.net/kmesh/pkg/controller/workload/bpfcache"
+)
+
+const (
+	xdpTailCallMap = "km_xdp_tailcall"
+)
+
+func testWorkload(t *testing.T) {
+	t.Run("XDP", testXDP)
+}
+
+func testXDP(t *testing.T) {
```

Comment:
I would suggest the name equals to that in xdp_authz_offload_test.c and xdp_shutdown_in_userspace_test.c

Reply:
So that we can maintain easily in future